pysindy/feature_library/base.py

```diff
@@ -487,32 +509,11 @@ def get_feature_names(self, input_features=None):
         -------
         output_feature_names : list of string, length n_output_features
         """
-        feature_names = list()
-        for i in range(len(self.libraries)):
-            lib_i = self.libraries[i]
-            if input_features is None:
-                input_features_i = [
-                    "x%d" % k for k in _unique(self.inputs_per_library[i])
-                ]
-            else:
-                input_features_i = np.asarray(input_features)[
-                    _unique(self.inputs_per_library[i])
-                ].tolist()
-            lib_i_feat_names = lib_i.get_feature_names(input_features_i)
-            for j in range(i + 1, len(self.libraries)):
-                lib_j = self.libraries[j]
-                if input_features is None:
-                    input_features_j = [
-                        "x%d" % k for k in _unique(self.inputs_per_library[j])
-                    ]
-                else:
-                    input_features_j = np.asarray(input_features)[
-                        _unique(self.inputs_per_library[j])
-                    ].tolist()
-                lib_j_feat_names = lib_j.get_feature_names(input_features_j)
-                feature_names += self._name_combinations(
-                    lib_i_feat_names, lib_j_feat_names
-                )
+        feature_names = self._feature_names_at(0, input_features)
```

Comment:
```suggestion
        check_is_fitted(self)

        if input_features is None:
            input_features = ["x%d" % i for i in range(self.n_features_in_)]
        input_features = np.array(input_features)
        inputs0 = input_features[self.inputs_per_library[0]]
        feature_names = self.libraries[0].get_feature_names(inputs0)
```

It's ok to require `get_feature_names()` be called after `fit()`, since the purpose of `fit()` for function libraries is just to determine `self.n_features_in_` and `self.n_features_out_`. `CustomLibrary` and several others already do this.